src/test/java/com/philips/research/spdxbuilder/persistence/ort/OrtReaderTest.java

```diff
@@ -13,21 +13,43 @@
 import java.nio.file.Path;
 import java.util.List;
 
+import static junit.framework.TestCase.assertTrue;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 
 class OrtReaderTest {
-    private static final File ORT_SAMPLE = Path.of("src", "test", "resources", "ort_sample.yml").toFile();
 
-    private final BillOfMaterials bom = new BillOfMaterials();
-
-    @Test
-    void loadsOrtSample() {
-        new OrtReader(ORT_SAMPLE)
-                .defineProjectPackage("NPM::mime-types:2.1.18", URI.create("pkg:npm/mime-types@2.1.18"))
-                .excludeScopes("NPM::mime-types:2.1.18", List.of("test*"))
-                .read(bom);
-
-        assertThat(bom.getPackages()).hasSize(1 + 2);
-    }
+  private static final File ORT_SAMPLE1 = getPathOfFile("ort_sample1.yml");
```

Comment:
Let's name the sample files after what they contain, so it is easier to recognize what they could (not) be used for.

E.g.: "ort_sample.yml" and "ort_with_issue.yml" (or something similar)